backend/nodes/image_nodes.py

```diff
@@ -48,7 +48,7 @@ def get_extra_data(self) -> Dict:
         img = self.result[0]
 
         if img.ndim == 2:
-            h, w, c = img.shape[:2], 1
```

Comment:
Did we use this pattern is other places too?

Reply:
No